java/cdk-environment/src/main/resources/nightlyTestSettings.json

```diff
@@ -1,4 +1,5 @@
 {
+  "deployId": "${deployId}",
```

Comment:
Is there any value have the id for the nightly tests tethered to the environment that they are run within for future debugging/investigation?

Reply:
Yes, that's what it's for :) how would you tether it? This file ends up in the build EC2 with the value you set at deploy time.